src/shapes/polyline.class.ts

```diff
@@ -14,18 +14,6 @@ import { degreesToRadians } from '../util/misc/radiansDegreesConversion';
 import { toFixed } from '../util/misc/toFixed';
 import { FabricObject, fabricObjectDefaultValues } from './Object/FabricObject';
 
-const strokeProjectionOptions: (keyof TProjectStrokeOnPointsOptions)[] = [
-  'scaleX',
-  'scaleY',
```

Comment:
@luizzappa did I merge this correctly

Reply:
This is the commit you removed it so I think it is good to go
https://github.com/fabricjs/fabric.js/commit/494a10ee2f8c2278ae9a55b20bf50cf6ee25b064

Reply:
I also tested visuals against 8d5f319acc9362860c495e8ec50b4c850b04e111 so I am happy